src/main/java/seedu/address/model/task/exceptions/DuplicateTaskException.java

```diff
@@ -0,0 +1,11 @@
+package seedu.address.model.task.exceptions;
+
+/**
+ * Signals that the operation will result in duplicate Tasks (Tasks are considered duplicates if they have the same
+ * identity).
+ */
+public class DuplicateTaskException extends RuntimeException {
```

Comment:
Can consider making this an checked exception (i.e. extend Exception class) instead since it's a recoverable error.